situp-plugins/elasticsearch/src/main/java/com/amazon/situp/plugins/sink/elasticsearch/ElasticsearchSink.java

```diff
@@ -119,18 +121,13 @@ private long calcEstimatedSizeInBytes(final IndexRequest indexRequest) {
 
   private boolean flushBatch(final BulkRequest bulkRequest) {
     try {
-      final BulkResponse bulkResponse = restHighLevelClient.bulk(bulkRequest, RequestOptions.DEFAULT);
-      final boolean hasFailures = bulkResponse.hasFailures();
-      // TODO: apply retry here
-      if (hasFailures) {
-        handleFailures(bulkRequest.requests(), bulkResponse.getItems());
-      }
-
-      return !hasFailures;
-    } catch (final IOException e) {
-      handleFailures(bulkRequest.requests(), e);
+      bulkRetryStrategy.execute(bulkRequest);
+    } catch (final InterruptedException e) {
+      LOG.error(e.getMessage(), e);
```

Comment:
Its a best practice to have a message with error logs. 

``` LOG.error("some message",e);```

Reply:
Here I do not have the exact message to describe about the interruption thrown by Thread.sleep, any suggestion?